tools/mksurfdata_map/src/mklaiMod.F90

```diff
@@ -335,13 +337,13 @@ integer,  pointer     :: laimask(:,:)     ! lai+sai output mask for each plant f
      do l = 0, numpft_i - 1
      do no = 1,ns_o
         glai_o(l)  = glai_o(l) + mlai_o(no,l)*tgridmap%area_dst(no)* &
-             tgridmap%frac_dst(no)*re**2
+             wtnorm(no)*re**2
```

Comment:
Although wtnorm made sense as an internal variable name in the routines in gridmapmod, when you take it outside that context, here and elsewhere, I feel that name isn't self-describing enough. I'd suggest changing this to something like `frac_dst` wherever it's used in the code.